js/utils.ts

```diff
@@ -0,0 +1,20 @@
+import type { AnyModel } from "@anywidget/types";
```

Comment:
>  Figure out how to simplify font loading.

I was looking into this [Font Loading API](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Font_Loading_API) which lets you add fonts to the global font index at the document level, and uses JS to kick off the font load. Maybe we can add an utility method here, `setupRender()`,  where we can call before every render method to standardize them? Or just `loadFonts()` would work!

```js
function loadFonts() {
  const font = new FontFace("Material Symbols Outlined", 'url(https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined)');
  document.fonts.add(font);
  font.load();
  return document.fonts.ready;
}
```